src/foundry/foundry.js/pixi/containers/placeableObjects/wall.d.ts

```diff
@@ -136,7 +136,11 @@ declare global {
     protected _onCreate(): void;
 
     /** @override */
-    protected _onUpdate(changed: DeepPartial<BaseWall['data']>, options?: any, userId?: string): void;
+    protected _onUpdate(
+      changed: DeepPartial<InstanceType<ConfiguredDocumentClass<typeof WallDocument>>['data']>,
```

Comment:
what's the point of this change? The data shouldn't be different between the base document and configured one. We could probably also just use `foundry.data.WallData` instead...?